src/File/FileApi.php

```diff
@@ -46,12 +46,12 @@ public static function create(AuthApiInterface $authProvider, $projectId, $logge
      */
     protected function processBodyOptions($requestData = []) {
         $opts = parent::processBodyOptions($requestData);
-        $key = 'file';
 
         if (!empty($opts['multipart'])) {
             foreach ($opts['multipart'] as &$data) {
-                if ($data['name'] == $key) {
+                if ($data['name'] === 'file') {
                     $data['contents'] = $this->readFile($data['contents']);
+                    $data['filename'] = array_key_exists('name', $opts) ? $opts['name'] : self::FILE_NAME_NOT_SET;
```

Comment:
I thought the fixx will be something like this https://github.com/Smartling/api-sdk-php/pull/121/files

Reply:
Discussed, leaving fix as is for the time being